ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -714,10 +716,12 @@ export class FrontstageDef {
     if (!widgetDef) return false;
 
     const popoutContent = (
-      <ChildWindowWidget
-        widgetContainerId={widgetContainerId}
-        widgetDef={widgetDef}
-      />
+      <ErrorBoundary FallbackComponent={WidgetFallback}>
```

Comment:
I'd move the `ErrorBoundary` to `ChildWindowWidget`.

Reply:
At least this way I get that `ChildWindowWidget` has no child property, so it can't nest elements.
![image](https://github.com/user-attachments/assets/68e1cf9f-b70c-4d4c-9f5e-e32e553b0b5e)

Reply:
You are trying to pass `ErrorBoundary` as a `children` prop to `ChildWindowWidget`, instead move it to `ChildWindowWidget.tsx`